js/samples/08.datasource.azureAISearch/appPackage/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Above: use different icon for sample manifest

Reply:
updated